### DISCUSSION TITLE
Ne plus recevoir de nombreux courriels automatiques

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1672] [Idées]
**Problème :**
Les gestionnaires de multiples cantines reçoivent autant de courriels automatiques concernant le diagnostic qu'ils gèrent de cantines. Mais quand ils gèrent plusieurs dizaines, voire plusieurs centaines, de cantines cela signifie plusieurs dizaines à centaines de courriels dans leur boîte électronique.

**Solutions envisagées :**
- Plutôt que d'envoyer un courriel automatique par cantine et par gestionnaire, envoyer plutôt un courriel uniquement par gestionnaire en sorte que la personne reçoive un courriel listant toutes les cantines concernées par le diagnostic manquant. Cette solution serait utile pour les gestionnaires de multiples cantines qui comptent remplir des données dans le diagnostic sans pour autant qu'ils se sentent spammés. Ex. : certains référents restauration en Département.
- Avoir une case "Envoi de courriels automatiques", que l'on pourrait cocher ou non, sur la fiche des gestionnaires. Cette solution serait utile pour les gestionnaires de multiples cantines qui veulent avoir accès aux cantines, mais ne comptent pas remplir de données dans les fiches des cantines. On ne cocherait pas la case pour ces gestionnaires. Ex. : d'autres référents restauration en Département.

Qu'en pensez-vous, @JenniferStephan, @hfroot et @alemangui ? Avez-vous d'autres solutions à l'esprit?

**Comment by alemangui** [comment]
Bonne remarque ! Je pense qu'on peut faire les deux :
- Agréger les emails de sorte que l'utilisateur en reçoive moins (même si un peu compliqué car on envoi ces emails à une semaine et deux semaines de la création de la cantine, donc le groupement est limité)
- Donner la possibilité aux utilisateurs de dire "-[ ] Je ne veux pas des emails reminder". Ça c'est simple, on peut le mettre en place rapidement.

**Reply by ghost** [reply]
Cool! Merci, @alemangui.

**Reply by alemangui** [reply]
Pour info, avec [cette PR](https://github.com/betagouv/ma-cantine/pull/1695) on peut désactiver les emails automatiques par utilisateur dans l'admin. Une prochaine étape pourrait être de permettre à l'utilisateur même de faire cette modification depuis leur profile.

Pour l'instant, dans les templates des emails automatiques on pourrait rajouter un petit texte à la fin du style « _Si vous ne voulez pas recevoir des emails comme celui-ci, contactez-nous_ »

**Reply by ghost** [reply]
Merci de l'info, @alemangui.

**Comment by ghost** [comment]
La case à cocher "Désactiver les notifs" sur admin ne permet pas de répondre à un cas : celui où l'utilisateur souhaite recevoir un rappel pour la télédéclaration, mais pas autant de rappels qu'il gère de cantines. Je pense qu'un seul courriel de rappel qui concerne toutes les cantines gérées par l'utilisateur permettrait de répondre à ce cas.

@alemangui @hfroot @JenniferStephan
